ci/etcd-manifest.yml

```diff
@@ -0,0 +1,58 @@
+name: phalanx-srini
```

Comment:
Merge the new etcd stuff into the existing CI manifest.  Otherwise, these tests will not be run.